lib/staccato/v4.rb

```diff
@@ -37,10 +37,11 @@ def self.default_adapter
     end
 
     # JSON encode in the case of GA measurement protocol V4
-    def self.encode_body(client_id, user_id, events)
+    def self.encode_body(client_id, user_id, timestamp_micros, events)
```

Comment:
We're changing the method signature of positional arguments and, particularly where we're operating a fork, that feels disruptive.

Since this API / module is forked instead of homegrown, I'm disinclined to refactor it to use keyword arguments, which, I think is the better answer here.

Instead, we should preserve the existing positional arguments along the following lines:

```ruby
def self.encode_body(client_id, user_id, events, timestamp_micros = nil)
```